cmd/reporter/app/cmd_event.go

```diff
@@ -51,7 +51,12 @@ func eventCommandHandler(cmd *cobra.Command, args []string) error {
 	}
 
 	ctx := context.Background()
-	return tracker.Tracker.SendEvent(ctx, event)
+
+	tracker, err := tracker.NewTracker()
```

Comment:
Since package name is tracker and variable name is also tracker.
Can you please rename variable name ?

```
t, err := tracker.NewTracker()
t.SendEvent
```